readme.md

```diff
@@ -1,11 +1,11 @@
-Street Facing - [Demo](http://alexhancock.github.com/street-facing/)
```

Comment:
Seems that those are already https links - [see this commit](https://github.com/alexhancock/street-facing/pull/2/commits/7e615f0b8b67e6253063a8d885ad0b84037c3a2f) - you may not be on the HEAD.

You might wanna switch them back to your gh-pages once merged anyway.